modules/ducktests/tests/ignitetest/services/ignite.py

```diff
@@ -86,4 +86,4 @@ def get_event_time(service, log_node, log_pattern, from_the_beginning=True, time
         "grep '%s' %s" % (log_pattern, IgniteAwareService.STDOUT_STDERR_CAPTURE))
 
     return datetime.strptime(re.match("^\\[[^\\[]+\\]", stdout.read().decode("utf-8")).group(),
-                             "[%Y-%m-%d %H:%M:%S,%f]")
+                             "[%Y-%m-%dT%H:%M:%S,%f]")
```

Comment:
why changed?

Reply:
format log4j2
example output:
[2020-12-09T19:10:58,878][INFO ][main][IgniteAwareApplicationService] Starting Application... [params=/mnt/service/ignite-config.xml]

Reply:
no, my question is "why the format changed"

Reply:
format ISO8601:
https://en.wikipedia.org/wiki/ISO_8601

before it was not applied

Reply:
From your link:

>  single point in time can be represented by concatenating a complete date expression, the letter "T" as a delimiter, and a valid time expression. For example, "2007-04-05T14:30". In ISO 8601:2004 it was permitted to omit the "T" character by mutual agreement as in "200704051430", but this provision was removed in ISO 8601-1:2019. **Separating date and time parts with other characters such as space is not allowed in ISO 8601, but allowed in its profile RFC 3339**.

As I know RFC 3339 is wide adopted too. Actually do we need care about it?